samples/bidirectional-chat/csharp/Function.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.WebJobs.Extensions.SignalRService;
+using Microsoft.AspNetCore.SignalR;
+
+namespace bidirectional_chat
+{
+    public class SimpleChat : ServerlessHub
+    {
+        private const string Hub = "simplechat";
+        private const string NewMessageTarget = "newMessage";
+        private const string NewConnectionTarget = "newConnection";
+
+        [FunctionName("negotiate")]
+        public static SignalRConnectionInfo GetSignalRInfo(
+            [HttpTrigger(AuthorizationLevel.Anonymous)] HttpRequest req,
+            [SignalRConnectionInfo(HubName = Hub, UserId = "{headers.x-ms-signalr-user-id}")] SignalRConnectionInfo connectionInfo)
+        {
+            return connectionInfo;
```

Comment:
looks like the "admin" claim doesn't send to RT

Reply:
Yes, I would like to add the claim by using `IdToken` and `ClaimTypeList` but now the binding expression doesn't work now.